source/adapters/cuda/memory.cpp

```diff
@@ -44,22 +44,21 @@ UR_APIEXPORT ur_result_t UR_APICALL urMemBufferCreate(
     CUdeviceptr Ptr = 0;
     auto HostPtr = pProperties ? pProperties->pHost : nullptr;
 
-    ur_mem_handle_t_::MemImpl::BufferMem::AllocMode AllocMode =
-        ur_mem_handle_t_::MemImpl::BufferMem::AllocMode::Classic;
+    BufferMem::AllocMode AllocMode = BufferMem::AllocMode::Classic;
```

Comment:
Lack of `ur_mem_handle_t_::MemImpl`, is because of the now gone union, right? Not some namespace mis-match?

Reply:
Correct